client/src/components/main/RecipeItem.jsx

```diff
@@ -5,42 +5,54 @@ import pascalCase from '../../utils/pascalCase';
 
 const RecipeItem = ({ recipe, upvote, downvote }) =>
   (<div className="col l6">
-    <div className="card views-small">
+    <div className="card hoverable views-small">
       <div className="card-image">
         <Link to={`/recipes/${recipe.id}`}>
           <img
-            src={recipe.image ? `../uploads/${recipe.image}`
-              : './css/img/spice.jpg'}
+            src={`../uploads/${recipe.image}`}
             alt={recipe.recipeName}
           />
         </Link>
         <span className="card-title">{recipe.views} Views</span>
       </div>
-      <div className="card-content small-cards">
-        <span className="card-title">
+      <div
+        style={{ display: 'flex', flexDirection: 'column' }}
+        className="card-content small-cards"
+      >
+        <div className="card-title">
           <strong>{recipe.recipeName}</strong>
-        </span>
-        <span
-          onClick={() => downvote(recipe.id)}
-          className="unlike right"
+        </div>
+        <div style={{
+          display: 'inline-block',
+          flexDirection: 'row',
+          marginTop: '-1rem'
+        }}
         >
-          <i
-            className="material-icons tiny reaction"
-          >thumb_down</i>{recipe.downvote}
-        </span>
-        <span
-          onClick={() => upvote(recipe.id)}
-          className="like right"
-        >
-          <i
-            className="material-icons tiny reaction"
-          >thumb_up</i>{recipe.upvote}
-        </span>
-        <Link
-          to={`/recipes/${recipe.id}`}
-          className="chip teal white-text"
-        >{pascalCase(recipe.category)}
-        </Link>
+          <Link
+            style={{ top: 'unset' }}
+            to={`/recipes/${recipe.id}`}
+            className="chip teal white-text"
+          >{pascalCase(recipe.category)}
+          </Link>
+          <span>
+            <div
+              onClick={() => downvote(recipe.id)}
+              className="unlike right"
```

Comment:
Using anonymous functions inside an onClick event listener introduces performance issue to your app relative to memory management.

Because every time component re-renders, an new memory is allocated for the anonymous function.

Consider using `downvote.bind(null, recipe.id)` instead.